myia/parser.py

```diff
@@ -958,6 +958,20 @@ def _process_Assign(self, block, node):
 
         return block
 
+    def _process_AugAssign(self, block, node: ast.AugAssign):
+        # Replace `a {op}= b` with `a = a {op} b`
```

Comment:
That's not how Python translates it. `a += b` translates to `a = operator.iadd(a, b)`.